Helper/Order.php

```diff
@@ -974,6 +977,23 @@ public function saveUpdateOrder($reference, $storeId = null, $boltTraceId = null
         return [$parentQuote, $order];
     }
 
+
+    /**
+     * @param $transaction
+     * @param $order
+     * @return bool
+     * @throws \Exception
+     */
+    public function isCreditHookIgnored($transaction, $order)
+    {
+        $currencyCode = $order->getOrderCurrencyCode();
+        $refundAmount = $transaction->amount->amount;
+        $boltTotal = $transaction->order->cart->total_amount->amount;
+        $magentoTotal = CurrencyUtils::toMinor($order->getGrandTotal(), $currencyCode);
+
+        return $boltTotal - $refundAmount == $magentoTotal;
+    }
+
```

Comment:
Does this approach works with two price change / credit requests of the same amount (50% of the total each)?Say, first the price change from the admin (as in the task) and then the refund of the rest.